werkzeug/exceptions.py

```diff
@@ -520,6 +520,20 @@ class Locked(HTTPException):
     )
 
 
+class Depended(HTTPException):
```

Comment:
This should match the error: `FailedDependency`

Reply:
I can't see such line in the code

Reply:
You wrote `Depended`, which doesn't match the exception title, which is "Dependency Failed".

Reply:
Ok, got it. Will fix that. Thanks